be/src/runtime/datetime_value.cpp

```diff
@@ -1535,7 +1521,10 @@ bool DateTimeValue::unix_timestamp(int64_t* timestamp, const std::string& timezo
     if (!find_cctz_time_zone(timezone, ctz)) {
```

Comment:
can ctz put it into cache like 'data foramt' function?

Reply:
It needs a thread safe cache.
Since there is not much calls of `find_cctz_time_zone()`, I think we can improve it later.